packages/webdriverio/src/utils/interception/devtools.ts

```diff
@@ -207,12 +239,12 @@ export default class DevtoolsInterception extends Interception {
      * Abort the request once with an error code
      * @param {string} errorReason  error code of the response
      */
-    abortOnce (errorReason) {
+    abortOnce (errorReason: string) {
         this.abort(errorReason, false)
     }
 }
 
-const filterMethod = (method, expected) => {
+const filterMethod = (method: string, expected?: ((method: string) => boolean) | string) => {
```

Comment:
We can create an expected type that looks like this:

```ts
type ExpectParameter<T> = ((param: T) => boolean) | T
```

and then use it throughout the file like this:
```suggestion
const filterMethod = (method: string, expected?: ExpectParameter<string>) => {
```

Reply:
Good suggestion! 👍 Changed!